four-principles.md

```diff
@@ -0,0 +1,10 @@
+Operate First regroups a few concepts, relying on standard definitions of Open, Open Source, and Open Work:
+
+OPERATE:
+We build software optimized to run as a service and allow others to run it as a service easily, including documenting how we operate the cloud environment itself.
+
+OPEN: We develop open source practices, tooling, and documentation that allows anyone to build and manage our software, incorporate lessons we’ve learned from operations back into code, automate wherever possible, and contribute to open source communities.
+
+GOVERNANCE: We foster inclusive, respectful, and collaborative communities through clear standards and accepted practices (e.g., codes of conduct) so participants are empowered to contribute.
+
+HYBRID: We ensure that there is no dependency in our tooling to the cloud environment we are operating in by always verifying our software and management tooling works identically on at least one other platform (another cloud, in a private data center, …)
```

Comment:
```suggestion
HYBRID: To ensure there is no dependency in the tooling of the cloud environment operating in by always verifying the software and management tooling works identically on at least one other platform (another cloud, in a private data center, …)
```

Reply:
Sentence reads a little awkardly to me. Suggest removing the "operating in" or adding more to it, like "...the software is operating in"